### PR TITLE
Wfs spatial indexes

### PR DESCRIPTION
**Pull request by emalfroy**

### CODE REVIEW

File: src/BuildingRegistry.Projections.Wfs/Migrations/20230825093350_AddSpatialIndexesV2.cs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+namespace BuildingRegistry.Projections.Wfs.Migrations
+{
+    public partial class AddSpatialIndexesV2 : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.Sql(@"
+	            ALTER TABLE [wfs].[buildingsV2]
+		            ADD CalculatedGeometry AS (geometry::STGeomFromWKB([Geometry], 31370)) PERSISTED
+	            GO");
```
Comment (ArneD) [inline src/BuildingRegistry.Projections.Wfs/Migrations/20230825093350_AddSpatialIndexesV2.cs:14]: why?

File: src/BuildingRegistry.Projections.Wfs/Migrations/20230825093350_AddSpatialIndexesV2.cs [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+			            LEVEL_2 = MEDIUM,
+			            LEVEL_3 = MEDIUM,
+			            LEVEL_4 = MEDIUM),
+	            CELLS_PER_OBJECT = 5)
+	            GO");
+
+            migrationBuilder.Sql(@"
+	            ALTER TABLE [wfs].[buildingUnitsV2]
+		            ADD CalculatedGeometry AS (geometry::STGeomFromWKB([Position], 31370)) PERSISTED
+	            GO");
```
Comment (ArneD) [inline src/BuildingRegistry.Projections.Wfs/Migrations/20230825093350_AddSpatialIndexesV2.cs:32]: same question

File: src/BuildingRegistry.Projections.Wfs/Migrations/20230825093350_AddSpatialIndexesV2.cs [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.Sql(@"
+	            ALTER TABLE [wfs].[buildingsV2]
+		            ADD CalculatedGeometry AS (geometry::STGeomFromWKB([Geometry], 31370)) PERSISTED
+	            GO");
+
+            migrationBuilder.Sql(@"
+	            CREATE SPATIAL INDEX [SPATIAL_GebouwV2_Geometrie] ON [wfs].[buildingsV2] ([CalculatedGeometry])
```
Comment (ArneD) [inline src/BuildingRegistry.Projections.Wfs/Migrations/20230825093350_AddSpatialIndexesV2.cs:17]: `[Geometry]`

File: src/BuildingRegistry.Projections.Wfs/Migrations/20230825093350_AddSpatialIndexesV2.cs [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+	            CELLS_PER_OBJECT = 5)
+	            GO");
+
+            migrationBuilder.Sql(@"
+	            ALTER TABLE [wfs].[buildingUnitsV2]
+		            ADD CalculatedGeometry AS (geometry::STGeomFromWKB([Position], 31370)) PERSISTED
+	            GO");
+
+            migrationBuilder.Sql(@"
+                CREATE SPATIAL INDEX [SPATIAL_GebouweenheidV2_Geometry] ON [wfs].[buildingUnitsV2] ([CalculatedGeometry])
```
Comment (ArneD) [inline src/BuildingRegistry.Projections.Wfs/Migrations/20230825093350_AddSpatialIndexesV2.cs:35]: `[Geometry]`

**Review by ArneD — CHANGES_REQUESTED** [review]

File: src/BuildingRegistry.Projections.Wfs/Migrations/20230825093350_AddSpatialIndexesV2.cs (hunk 3 above)
Comment (emalfroy) [reply src/BuildingRegistry.Projections.Wfs/Migrations/20230825093350_AddSpatialIndexesV2.cs]: copy pasta from wms. did not notice it was already Geometry

File: src/BuildingRegistry.Projections.Wfs/Migrations/20230825093350_AddSpatialIndexesV2.cs (hunk 4 above)
Comment (emalfroy) [reply src/BuildingRegistry.Projections.Wfs/Migrations/20230825093350_AddSpatialIndexesV2.cs]: with buildingUnits its from [Position]

File: src/BuildingRegistry.Projections.Wfs/Migrations/20230825093350_AddSpatialIndexesV2.cs [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+#nullable disable
+
+namespace BuildingRegistry.Projections.Wfs.Migrations
+{
+    public partial class AddSpatialIndexesV2 : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.Sql(@"
+	            CREATE SPATIAL INDEX [SPATIAL_GebouwV2_Geometrie] ON [wfs].[buildingsV2] [Geometry]
```
Comment (ArneD) [inline src/BuildingRegistry.Projections.Wfs/Migrations/20230825093350_AddSpatialIndexesV2.cs:12]: ([Geometry]) same below

**Review by ArneD — CHANGES_REQUESTED** [review]

File: src/BuildingRegistry.Projections.Wfs/Migrations/20230825093350_AddSpatialIndexesV2.cs (hunk 5 above)
Comment (emalfroy) [reply src/BuildingRegistry.Projections.Wfs/Migrations/20230825093350_AddSpatialIndexesV2.cs]: done

**Review by ArneD — APPROVED** [review]

**Pull request merged**